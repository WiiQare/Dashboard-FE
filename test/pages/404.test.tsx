import { render } from "@testing-library/react";
import Error404 from "../../src/pages/404";

test("renders Error404 component", async () => {
    const { container } = render(<Error404 />);
    expect(container).toMatchSnapshot();
});
