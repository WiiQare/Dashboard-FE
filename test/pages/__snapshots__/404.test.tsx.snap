// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Error404 component 1`] = `
<div>
  <div
    class=" min-h-screen w-screen bg-blue-100 flex items-center p-5 lg:p-[20rem] overflow-hidden relative"
  >
    <div
      class="flex-1 min-h-full min-w-full rounded-3xl bg-white shadow-xl p-10 lg:p-20 text-gray-800 relative md:flex items-center text-center md:text-left"
    >
      <div
        class="w-full md:w-1/2"
      >
        <div
          class="mb-10 md:mb-20 text-gray-600 font-light"
        >
          <h1
            class="font-black uppercase text-3xl lg:text-5xl text-orange mb-10"
          >
            You seem to be lost!
          </h1>
          <p>
            The page you're looking for isn't available.
          </p>
          <p>
            Try searching again or use the Go Back button below.
          </p>
        </div>
        <div
          class="mb-20 md:mb-0"
        >
          <a
            class="text-lg font-light outline-none focus:outline-none transform transition-all hover:scale-110 text-orange hover:text-yellow-600 flex items-center gap-1"
            href="/"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M18.464 2.114a.998.998 0 0 0-1.033.063l-13 9a1.003 1.003 0 0 0 0 1.645l13 9A1 1 0 0 0 19 21V3a1 1 0 0 0-.536-.886zM17 19.091 6.757 12 17 4.909v14.182z"
              />
            </svg>
             Go Back
          </a>
        </div>
      </div>
      <div
        class="w-full md:w-1/2 text-center"
      >
        <a
          class="text-xs text-gray-300"
        >
          WiiQare Ã  votre service
        </a>
      </div>
    </div>
    <div
      class="w-64 md:w-96 h-96 md:h-full bg-blue-200 bg-opacity-30 absolute -top-64 md:-top-96 right-20 md:right-32 rounded-full pointer-events-none -rotate-45 transform"
    />
    <div
      class="w-96 h-full bg-yellow-200 bg-opacity-20 absolute -bottom-96 right-64 rounded-full pointer-events-none -rotate-45 transform"
    />
  </div>
</div>
`;
